Source/WebCore/dom/Element.cpp

```diff
@@ -4510,6 +4510,13 @@ bool Element::hasKeyframeEffects(PseudoId pseudoId) const
     return false;
 }
 
+bool Element::hasAnimations() const
+{
+    if (!hasRareData())
+        return false;
+    return elementRareData()->hasAnimations();
```

Comment:
nit: could be a one liner:
`return hasRareData() && elementRareData()->hasAnimations()`

Reply:
Fixed.